packages/db/src/query/live-query-collection.ts

```diff
@@ -163,6 +163,12 @@ export function liveQueryCollectionOptions<
   const collections = extractCollectionsFromQuery(query)
 
   const allCollectionsReady = () => {
+    return Object.values(collections).every(
+      (collection) => collection.status === `ready`
```

Comment:
Collection's have a method that checks exactly this: `collection.isReady()`. Better to use that one.